src/earn/EarnEnterAmount.test.tsx

```diff
@@ -416,7 +402,49 @@ describe('EarnEnterAmount', () => {
       expect(queryByTestId('downArrowIcon')).toBeFalsy()
     })
 
+    it('should prepare transactions with the expected inputs', async () => {
+      const { getByTestId } = render(
+        <Provider store={store}>
+          <MockedNavigator
+            component={EarnEnterAmount}
+            params={{ pool: mockPoolWithHighPricePerShare, mode: 'withdraw' }}
+          />
+        </Provider>
+      )
+
+      fireEvent.changeText(getByTestId('EarnEnterAmount/TokenAmountInput'), '.25')
+
+      await waitFor(() => expect(refreshPreparedTransactionsSpy).toHaveBeenCalledTimes(1))
+      expect(refreshPreparedTransactionsSpy).toHaveBeenCalledWith({
+        amount: '0.125',
+        token: {
+          ...mockTokenBalances[mockAaveArbUsdcTokenId],
+          priceUsd: new BigNumber(1),
+          lastKnownPriceUsd: new BigNumber(1),
+          balance: new BigNumber(10),
+        },
+        walletAddress: mockAccount.toLowerCase(),
+        pool: mockPoolWithHighPricePerShare,
+        hooksApiUrl: networkConfig.hooksApiUrl,
+        feeCurrencies: mockFeeCurrencies,
+        shortcutId: 'withdraw',
+        useMax: false,
+      })
+    })
+
     it('should show tx details for withdrawal', async () => {
+      jest.clearAllMocks()
```

Comment:
nit: remove the jest.clearAllMocks from individual tests, we have one at the top level describe.